client/libs/core-data/src/lib/users/users.service.ts

```diff
@@ -1,11 +1,13 @@
+/// <reference path="../../../typings.d.ts" />
+
 import { Injectable } from '@angular/core';
 import { HttpClient } from '@angular/common/http';
 
 import { Observable } from 'rxjs';
 
 import { User } from './user.model';
 
-const BASE_URL = 'http://localhost:8080/api/v1';
+const BASE_URL = `${process.env.ENDPOINT}/api/v1`;
```

Comment:
Angular won't have access to `process.env.ENDPOINT` at runtime.

Reply:
Correct...but it does at build time thanks to a custom Angular builder that leverages webpack.
This is the tut I followed: https://medium.com/@fidelisclayton/system-environment-variables-in-angular-1f4a922c7b4c